ui/app/src/components/ConfirmDialog/ConfirmDialog.tsx

```diff
@@ -27,6 +27,8 @@ export function ConfirmDialog(props: ConfirmDialogProps) {
   const { formatMessage } = useIntl();
   return (
     <AlertDialog
+      disableBackdropClick
```

Comment:
@rart I think that the confirm dialog should never be dismissed unless clicking one of the actions, what do you think?